app/livechat/server/lib/Helper.js

```diff
@@ -397,6 +397,7 @@ export const forwardRoomToDepartment = async (room, guest, transferData) => {
 	}
 
 	const { departmentId } = transferData;
+	const department = LivechatDepartment.findOneById(departmentId);
```

Comment:
Since this variable is only getting used within the if condition below, so I'd recommened moving it within that statement... just to avoid querying to DB when its not required